includes/sanitizers/class-amp-iframe-sanitizer.php

```diff
@@ -139,10 +160,27 @@ public function sanitize() {
 	private function normalize_attributes( $attributes ) {
 		$out = array();
 
+		$remove_allow_same_origin = false;
```

Comment:
Actually, now I wonder why `allow-same-origin` is even the default `sandbox`. AMP never allows iframes to mutate the parent DOM, so why is this ever useful?

Reply:
One key reason is the document would not be able to access cookies:

> Uncaught DOMException: Failed to read the 'cookie' property from 'Document': The document is sandboxed and lacks the 'allow-same-origin' flag.

Reply:
Another side effect of this is the `origin` in `postMessage` will be `null`.

Still, it seems far preferable to render an iframes with these sandbox restrictions than to render a blank box with an amp-iframe error.